specification/swlifecycle.adoc

```diff
@@ -25,13 +25,24 @@ controls described earlier).
 confidential memory regions for various purposes like creating TVM workloads
 etc.
 
-The host OS/VMM may create a new TVM by allocating and initializing a TVM
+[note]
+====
+In deployment model 3 described in <<appendix_d> creation of the confidential
+memory region is statically done at boot time.
+====
+
+In deployment models 1 and 2, the host OS/VMM may create a new TVM by 
+allocating and initializing a TVM
 using the `sbi_covh_create_tvm()` function. An initial set of memory pages are
 granted to the TSM and tracked as TEE pages associated with that TVM from
 that point onwards until the TVM is destroyed via the `sbi_covh_destroy_tvm()`
-function.
+function. In deployment model 3, all TVMs start execution as a VM.
+To become a TVM, the VM request that the TSM change it into
+a TVM using the sbi_coveh_promote_TVM call. 
```

Comment:
Can the promotion be anytime during the run of VM or any certain points ?

Reply:
The section in the appendix describes this. 
"To reduce the complexity of the TSM implementation, TSM creates TVM as a result of a single operation triggered with the `sbi_covg_promote_to_tvm()` call. Specifically, OS/VMM initializes and starts a regular VM, which early during the boot process requests to be promoted to TVM. This call is reflected to TSM that copies TVM data, the page table configuration, and the vCPU state into confidential memory."

Reply:
Thanks for reviewing. I updated the description of the time when the promotion should take place in the section `Single-step TVM Creation` in `TVM Lifecycle` chapter.